src/lib/utils/index.ts

```diff
@@ -11,6 +11,9 @@ export const isTruthy = <T>(value: T): value is Truthy<T> => Boolean(value);
 /** With strict equality check (i.e. `===`) */
 export const filterUnique = <T>(array: T[]) => Array.from(new Set(array));
 
+export const areStringArraysEqual = (arr1: string[], arr2: string[]) =>
+  arr1.reduce((acc, curr) => acc + curr, '') === arr2.reduce((acc, curr) => acc + curr, '');
```

Comment:
This array comparison is incorrect in general case because it returns `true` for arrays `['abcd', 'ef']` and `['abc', 'def']`. Consider using `isEqual` from `lodash` or manual comparison of arrays lengths and strings in each of them.